README.md

```diff
@@ -30,4 +30,9 @@ this app is built with the following libraries/packages:
 - [AWS SDK](https://aws.amazon.com/sdk-for-go/) for AWS S3
 
 ### Future Works
-- add more tests for the app (s3 package)
\ No newline at end of file
+- Add more tests for the app (s3 package)
+- Add more features to the app:
+  - store the data in a database after reading from prometheus, related to [Example app: upload to prometheus](https://github.com/project-flotta/flotta-edge-example#future-works)
```

Comment:
What is the reason not to fetch this data directly from prometheus?

Reply:
As far as I understood (from search), Prometheus sends a stream of data that I can fetch and present (I mean it can't be persisted), and if I want to analyze data I need to store some old data to perform some analysis, Do I miss something?

Reply:
please take a look [here](https://project-flotta.io/documentation/v0_2_0/operations/observability.html)